app/src/main/java/com/orange/ods/app/ui/AppBarActions.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.app.ui
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.TextFieldValue
+import com.orange.ods.app.R
+import com.orange.ods.app.ui.utilities.UiString
+import com.orange.ods.app.ui.utilities.extension.isDarkModeEnabled
+import com.orange.ods.compose.component.appbar.top.OdsTopAppBarActionButton
+import com.orange.ods.compose.component.appbar.top.OdsTopAppBarOverflowMenuActionItem
+import com.orange.ods.compose.component.content.OdsComponentContent
+import com.orange.ods.compose.component.textfield.search.OdsSearchTextField
+
+val alwaysVisibleActions = listOf(AppBarAction.ChangeTheme, AppBarAction.ChangeMode)
```

Comment:
As these actions are not always visible strictly speaking (on search screen for instance), should we rename this to `defaultActions`?

Reply:
why not